tools/src/main/java/com/nedap/archie/flattener/TerminologyFlattener.java

```diff
@@ -64,25 +64,7 @@ protected static ArchetypeTerm getTerm(Map<String, Map<String, ArchetypeTerm>> t
     private static void flattenValueSets(Map<String, ValueSet> childValueSets, Map<String, ValueSet> resultValueSets) {
         for(String key:childValueSets.keySet()) {
             ValueSet childValueSet = childValueSets.get(key);
-            ValueSet overriddenValueSet = findMatchingValueSet(resultValueSets, key);
```

Comment:
This was a really complicated way of ensuring no new values ever get added. However, that is done in the validator, and should not be repeated here.